util/chplenv/third_party_utils.py

```diff
@@ -236,11 +258,15 @@ def pkgconfig_get_bundled_link_args(pkg, ucp='', pcfile='',
     if d == None:
         return ([ ], [ ])
 
-    if 'Requires' in d and d['Requires']:
+    if d.get('Requires') and _pkgconfig_should_warn_for_requires(d):
         warning("Simple pkg-config parser does not handle Requires")
         warning("in {0}".format(pcpath))
 
-    if static and 'Requires.private' in d and d['Requires.private']:
+    if (
+        static
+        and d.get("Requires.private")
+        and _pkgconfig_should_warn_for_requires(d, private=True)
+    ):
```

Comment:
Trivial, but seems like really weird formatting. I would expect this:

```
if (static
    and d.get("Requires.private")
    and _pkgconfig_should_warn_for_requires(d, private=True)):
```

(But if there is some Python style convention that applies in this case, I wouldn't know about it).

Reply:
I used `black` to format the code, but I think both formattings match the PEP8 style guide

Reply:
Alright! Thanks for checking.